docs/SPIR-V.rst

```diff
@@ -1923,6 +1939,81 @@ Example 3: (compiled with ``-fvk-bind-globals 2 1``)
 Note that if the developer chooses to use this command line option, it is their
 responsibility to provide proper numbers and avoid binding overlaps.
 
+ResourceDescriptorHeaps & SamplerDescriptorHeaps
+------------------------------------------------
+
+The SPIR-V backend supported SM6.6 resource heaps, using 2 extensions:
+- `SPV_EXT_descriptor_indexing`
```

Comment:
I don't see this added anywhere explicitly in this PR, so just checking, is it added unconditionally by the capability visitor? Assuming it is, can you add some CHECKs for it in tests after capability stripping?

Reply:
Its added by DXC as soon as you have a resource of type OpRuntimeArray. So this PR isn't the first to use this extension, it's just another case where this extension is required.